docs/accessibility/guides/local-development.mdx

```diff
@@ -0,0 +1,26 @@
+---
+title: Feedback during local development
+sidebar_position: 50
+---
+
+# Feedback during local development
+
+Eventually, the Cypress App will support displaying accessibility feedback during local development. For now, the important thing to note is: you do _not_ need to push your changes and wait for your entire build process to complete.
+
+The best solution is to record to Cypress Cloud from your local development machine. This allows you to run a small set of tests directly related to your changes and fully skip your build pipeline and CI processes. It can be run with a one line command:
+
+```
+npx cypress run --key <record_key> --record --spec "cypress/e2e/my-spec.cy.js"
+```
+
+(You can skip the `--key` option by setting `CYPRESS_RECORD_KEY` as an environment variable. Learn more about recording in our [docs about recorded runs](https://docs.cypress.io/guides/cloud/recorded-runs)).
+
+This has 3 key benefits:
+
+- Accuracy \- you will see exactly what Cypress does with your new code changes, all config and software versions will match
```

Comment:
I don’t understand this “all config and software versions will match” statement

Reply:
Updated to clarify this is about using your Cloud AQ configuration values and the Cloud's version of Axe Core to avoid any discrepancy in results that might be there if you tested locally using a browser extension or a plugin that was not aware of those things.